specs/txUtils.spec.ts

```diff
@@ -83,17 +83,17 @@ describe('txUtils tests', () => {
   })
 
   describe('.waitForCompletion', function() {
-    it('should return a failed tx for a dropped hash', async function() {
+    it('should return a dropped tx for a dropped hash', async function() {
       txUtils.TRANSACTION_FETCH_DELAY = 10
 
       const droppedTx = await txUtils.waitForCompletion(
-        '0x24615f57f5754f2479d6657f7ac9a56006d8d6f634c6955310a5af1c79f4969'
+        '0x24615f57f5754f2479d6657f7ac9a56006d8d6f634c6955310a5af1c79f4969',
+        10
```

Comment:
It is not necessary but we can put this in a constant as we have here https://github.com/decentraland/decentraland-eth/pull/31/files#diff-8a608492eb1548a81c5ca55bfe88229bR69